compiler/src/main/scala/org/scalajs/core/compiler/JSGlobalAddons.scala

```diff
@@ -128,6 +129,18 @@ trait JSGlobalAddons extends JSDefinitions
     def isJSSetter(sym: Symbol): Boolean =
       nme.isSetterName(sym.name) && sym.isMethod && !sym.isConstructor
 
+    /** Is this symbol a static field at the IR level? */
+    def isStaticField(sym: Symbol): Boolean = {
```

Comment:
Can we call this `isFieldStatic` to make it extra clear, that this does not verify whether `sym` is even a field?